src/components/infopopups/common/infopopup.css

```diff
@@ -147,6 +147,12 @@
     background: #f2f2f2;
     height: 100%;
 }
+.add-vertical-line {
```

Comment:
why did you create a new class instead of editing the existing one?  The only use case was for the popups vertical line, that is why it was in the infopopups css file... ??